hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedTableMetadata.java

```diff
@@ -547,6 +547,7 @@ private void verifyMetadataMergedRecords(HoodieTableMetaClient metadataMetaClien
         .withRequestedSchema(schema)
         .withDataSchema(schema)
         .withProps(new TypedProperties())
+        .withEnableOptimizedLogBlockScan(metadataTableWriteConfig.enableOptimizedLogBlocksScan())
```

Comment:
This should also be metadataConfig as we are reading from mdt table using metadataMetaClient